### ISSUE TITLE
Not working on this page

### ISSUE THREAD

**Issue by orschiro** [gilgol2016/auto-overlay-remover#1]
Dear @gilgol2016,

Thank you so much for your extension!

I installed it to get rid of the overlay produced on this page:

http://www.collaborationsuperpowers.com/podcasts/

![image](https://cloud.githubusercontent.com/assets/170947/16176061/277f6d16-3603-11e6-959d-b38d8a29c92e.png)

However, regardless of the settings, the overlay keeps appearing. 

Any ideas?

Thankfully,

~Robert

**Comment by gilgol2016** [comment]
Hi Robert,
Thank you for your feedback and I'm glad that you use it.
Because of the overlay doesn't have formal rule, standard or behavior it's
hard to remove all of them.
I'll check your site you sent me (and several others) which the popups
still occurs and try to find solution for it.
I hope I'll update it in the next few weeks.
One more time, thank for your e-mail.

Sincerely,
Gil 😎

**Comment by orschiro** [comment]
Dear Gil,

I very appreciate your responsiveness, thank you! :smile: 

~Robert

**Comment by gilgol2016** [comment]
Dear Robert,
Didn't forget you.
I updated the extension. 
Added the option to save specific sites/domains to start with a delay by a choice. 
Tested it with delay of 25 seconds and it worked like a charm (with the new version 1.0.6).
Try to add the site to 'Delayed start overlay list' in the icon popup and set the timer to 25 seconds and that's it.
Hope that it will solve more sites for you now.

thanks,
Gil :)
